server/server.go

```diff
@@ -66,17 +67,18 @@ func NewSlackHandler(basePath, appToken, secretToken string, l LogFunc) *SlackHa
 			res.Text(http.StatusNotFound, "Not found")
 			return nil
 		},
-		Log:      l,
-		basePath: basePath,
-		appToken: appToken,
+		Log:         l,
```

Comment:
We need to rename this to `Logf` to indicate that the function takes a Printf style formatting.